daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala

```diff
@@ -66,13 +64,28 @@ class ExceptionTest extends AnyWordSpec with Matchers with TableDrivenPropertyCh
             SValue.SRecord(tyCon, data.ImmArray.empty, new util.ArrayList()),
           )
         )
+    val arithmeticCon = data.Ref.Identifier.assertFromString(
+      "f1cf1ff41057ce327248684089b106d0a1f27c2f092d30f663c919addf173981:DA.Exception.ArithmeticError:ArithmeticError"
+    )
+    val fields = new util.ArrayList[SValue]()
+    fields.add(SValue.SText("ArithmeticError while evaluating (DIV_INT64 1 0)."))
+    val divZeroE =
+      SValue.SAny(
+        TTyCon(arithmeticCon),
+        SValue.SRecord(
+          arithmeticCon,
+          data.ImmArray(data.Ref.Name.assertFromString("message")),
+          fields,
+        ),
+      )
 
     val testCases = Table[String, SResult](
       ("expression", "expected"),
       ("M:unhandled1", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled2", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled3", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled4", SResultError(DamlEUnhandledException(e2))),
+      ("M:divZero", SResultError(DamlEUnhandledException(divZeroE))),
```

Comment:
This is already tested in  `daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala` .
One more, in a slightly different context, do not harm.

Reply:
Yeah I saw that. It seemed useful to have one case for this here but also happy to drop it if you prefer.

Reply:
:+1:

Reply:
meaning you can keep it.